nolooking/umbrel-app.yml

```
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: nolooking
+category: Lightning Node Management 
+name: nolooking
+version: "0.1.0"
+tagline: Use PayJoin to open channels for your lightning node.
+description: >-
+  Nolooking leverages Pay-to-Endpoint (payjoin) to negotiate a channel open for your lightning node, initiated by any BIP78 supporting wallet.
+
+  You can open multiple lightning nodes in a single payment, improving privacy, saving on transaction fees and wait time.
+
+  Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear all the benefits!
```

Markdown isn't supported or rendered in Umbrel's UI, so we can probably tweak it to:

```suggestion
  Read the article about Lightning Powered PayJoin: https://chaincase.app/words/lightning-payjoin to hear all the benefits!
```